src/realmModels/Observation.js

```diff
@@ -110,7 +110,12 @@ class Observation extends Realm.Object {
       rank_level: true
     },
     taxon_geoprivacy: true,
-    time_observed_at: true
+    time_observed_at: true,
+    user: {
```

Comment:
What's the reason we're storing a user object here? Afaik, we should *only* be storing observations that belong to the logged in user, so this feels redundant -- this would always store the same user for every observation.

Reply:
Good point. I originally included it there to query for users in useInfiniteExploreScroll. I removed it from the realm file and only included the user object in the query fields in useInfiniteExploreScroll.